microovn/ovn/environment.go

```diff
@@ -199,7 +199,7 @@ func cleanupPaths() error {
 	if len(errs) > 0 {
 		errs = append(
 			errs,
-			fmt.Errorf("failures occured during backup. Refusing to continue with data removal"),
+			fmt.Errorf("failures occured during backup. refusing to continue with data removal"),
```

Comment:
nit: same as mentioned above. Either retain capitalization of "inner" sentence, or turn this error to single sentence with comma instead of dot.